src/cr/cube/cube.py

```diff
@@ -602,7 +597,7 @@ def _measures(self) -> "_Measures":
         Provides access to count based measures and numeric measures (e.g. mean, sum)
         when available.
         """
-        return _Measures(self._cube_dict, self._all_dimensions, self._cube_idx_arg)
+        return _Measures(self._cube_response, self._all_dimensions, self._cube_idx_arg)
```

Comment:
Here we build `_Measures` without the "dimension inflation". Do we still need it ?

Reply:
actually self._all_dimensions has the inflation